lib/koans/10_map_sets.ex

```diff
@@ -0,0 +1,62 @@
+defmodule MapSets do
+  use Koans
+
+  @intro "My name is Set, MapSet."
+
+  @set MapSet.new([1, 2, 3, 4, 5])
+
+  koan "I am merely another collection but, you can perform some operations on me" do
+    new_set = MapSet.new(@set, fn x -> 3 * x end)
```

Comment:
@felipesere I like the idea. The Enum koans come in at number 13 (on master). I placed this before the Enums koans. Would it be OK to use the Enum function before it is introduced or maybe this koan should come after the Enum koans?